scripts/scil_evaluate_bundles_individual_measures.py

```diff
@@ -118,7 +141,32 @@ def main():
                     output_measures_dict[measure_name] = []
                 output_measures_dict[measure_name].append(
                     measure_dict[measure_name])
-
+    # add set stats if there are more than one bundle
+    if args.group_statistics:
```

Comment:
I like the idea of average but kind of weigthed by streamlines count.
But it is confusing (a bit) that min/max length are truly just min/max.

Also where are elongation, curl, curvature? Is that volontary?
You should document this part more.